owa/webpack.config.js

```diff
@@ -156,7 +156,7 @@ plugins.push(new CopyWebpackPlugin([{
 var webpackConfig = {
   quiet: false,
   entry: {
-	  app : `${__dirname}/app/js/openmrs-owa-idgen`,
+	  app : ['babel-polyfill', `${__dirname}/app/js/openmrs-owa-idgen`],
```

Comment:
any reason for this change?

Reply:
I added it to enable me use es6 async/await. It provides support for all stage-0 features as well

Reply:
where have we used es6 async/await?

Reply:
I used it [here](https://github.com/andela-dbamidele/openmrs-module-idgen/blob/eb84303d2d21bb691550b1b94e1bb8d0648b3294/owa/app/js/components/manage_autogeneration_option/helpers.js)

Reply:
The same helpers.js which is a duplicate right?

Reply:
Do you mean duplicates to `apiHelper`? They are doing pretty different things.

Reply:
We are going in circles I don't know why. what is the purpose of helpers.js ?

Reply:
Instead of making my api calls in my component's componentDidMount, I created the helper to do the api calls for all the values I'll be using in my component. 

I could have done exactly the same thing in my component and it result in a lot of lines.

Reply:
do we still need this?